lib/environment.ts

```diff
@@ -20,17 +20,14 @@ export const setAuthToken = async (token: string) => {
   console.log("Setting authToken: ", maskToken(token));
 
   if (process.env.VERCEL === "1") {
-    await fetch(
-      process.env.SALEOR_MARKETPLACE_REGISTER_URL as string,
-      {
-        method: "POST",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify({
-          auth_token: token,
-          marketplace_token: process.env.SALEOR_MARKETPLACE_TOKEN,
-        }),
-      },
-    );
+    await fetch(process.env.SALEOR_MARKETPLACE_REGISTER_URL as string, {
```

Comment:
You might want to throw an error if this env is undefined (I personally find it easier to debug the issue if I forgot to configure the end 🙈)


There's this nice package that you can use here: https://www.npmjs.com/package/ts-invariant

```suggestion
    invariant(process.env.SALEOR_MARKETPLACE_REGISTER_URL, "Env var SALEOR_MARKETPLACE_REGISTER_URL is not configured");
    await fetch(process.env.SALEOR_MARKETPLACE_REGISTER_UR, {
```

Reply:
Added